test/integration/index.js

```diff
@@ -40,29 +40,31 @@ const basicExample = () => {
     })
 
     it('should announce status changes using two alternately updated aria live regions', () => {
-      const flip = browser.$('div#ariaLiveA')
-      const flop = browser.$('div#ariaLiveB')
+      if (isChrome) {
```

Comment:
Will this mean that in Firefox we'll still see 'should announce status changes using two alternately updated aria live regions' appearing to have passed?

I'm wondering if we should either swap the order of these so that we only define the test if `isChrome` is true, or perhaps even doing something like

```js
(isChrome ? it : it.skip)('should announce status changes using two alternately updated aria live regions', () => {
```

That way the test would either not appear in the output, or would be explicitly listed as having been skipped.

Thoughts?

Reply:
Yeah this seems worth having a poke, but if it's too difficult this follows the same pattern elsewhere in the test suite when running into browser specific issues.

Reply:
@36degrees We've wrapped the tests in a conditional block. Your above code seems to work too but the linter was complaining about it so this seemed simpler.